elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -32,17 +32,17 @@ class PfeContentSet extends PFElement {
   get settings() {
     let settings = {};
     const variant = this.getAttribute("pfe-variant");
+    const theme = this.getAttribute("on");
 
-    if (variant === "primary") {
+    if (variant === "wind") {
       settings.variant = variant;
-      settings.color = "striped";
-    } else if (variant === "secondary") {
+    } else if (variant === "earth") {
       settings.variant = variant;
-      settings.color = "dark";
-    } else {
-      settings.color = "lightest";
     }
 
+    if (theme === "dark") {
```

Comment:
Any reason we might want to let an `on="light"` propagate?  Maybe for specificity or to override something?

Reply:
yes!

Reply:
@castastrophe I have changed the logic here.